stonk_market_mayhem2.py

```
@@ -0,0 +1,76 @@
+# BAD STOCK TRADING APP - DO NOT USE IN REAL LIFE
+
+import random
+
+stocks = {
+    "AAPL": 150,
+    "GOOG": 2800,
+    "TSLA": 720,
+    "AMZN": 3400
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)